docs/manual/developer_guide.adoc

```diff
@@ -969,6 +969,39 @@ This is an example of a check, written using the custom OVAL syntax, that checks
   </unix:file_object>
 ----
 
+===== Macros
+
+Jinja macros for OVAL checks are located in `/shared/macros-oval.jinja`. These currently include the following high-level macros:
+
+- `oval_sshd_config` -- check a parameter and value in the sshd configuration
+- `oval_grub_config` -- check a parameter and value in the grub configuration
+- `oval_check_config_file` -- check a parameter and value in a given configuration file
+
+Always consider reusing `oval_check_config_file` when creating new macros, it has some logic that will save you some time (e.g.: platform applicability).
+
+They also include several low-level macros which are used to build the high level macros:
+
+- set of low-level macros to build the OVAL checks for line in file:
+```
+oval_line_in_file_criterion
+oval_line_in_file_test
+oval_line_in_file_object
+oval_line_in_file_state
+```
+
+- set of low-level macros to build the OVAL checks to test if a file exists:
+```
+oval_config_file_exists_criterion
+oval_config_file_exists_test
+oval_config_file_exists_object
+```
+
+====== Platform applicability
+Platform applicability is given by the `prodtype` property in the rule.yml file. If you are using `oval_check_config_file` macro directly or indirectly, it should be enough to define `prodtype`. Default is `all` platforms. If you intend to define your own OVAL check structure please consider using `oval_affected` macro from `/shared/macros.jinja`.
```

Comment:
`define your own OVAL check structure please` what do you mean by structure?

Reply:
I guess just removing `structure` from the sentence solves my confusion.

Reply:
done.